main.go

```diff
@@ -42,6 +44,11 @@ import (
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
+	Version  = "v0.1.0-dev"
```

Comment:
should this just be "dev" do you think? this is the version that would show up if `go get` was used to download the binary.

Reply:
Sure thing, I initially had it this way to test the semver comparison.

Reply:
should we disable update checking if the verison is dev?